pool/pool.go

```diff
@@ -380,6 +451,166 @@ func (p *pool) AnnounceContainerUsedSpace(ctx context.Context, announce []contai
 	return conn.AnnounceContainerUsedSpace(ctx, announce, options...)
 }
 
+func (p *pool) checkSessionTokenErr(err error, address string) {
+	if err == nil {
+		return
+	}
+
+	if strings.Contains(err.Error(), "session token does not exist") {
+		p.cache.DeleteByPrefix(address)
+	}
```

Comment:
Shouldn't pool retry request in case of this error after obtaining new session token? 

Consider storage node has been restarted but still considered to be healthy. Pool processes `PutObjectParam`, gets `session token does not exist`. After that pool can create one more session token, cache it and repeat request with more chances to success, isn't it?